client/my-sites/plans/jetpack-plans/use-detect-window-boundary.ts

```diff
@@ -2,9 +2,23 @@
  * External dependencies
  */
 import { useEffect, useRef, useState } from 'react';
-import { throttle } from 'lodash';
 
-export const THROTTLE_RATE = 50;
+const THROTTLE_RATE_MS = 50;
+
+const throttle = ( func, waitForMilliseconds ) => {
```

Comment:
😅 

So the problem with introducing `throttle` inline is that we'll have to do it plenty more times in the Calypso codebase - to be precise, 17 times (looking at `trunk`). Ideally, we should find a way to do it in a more reusable way. Perhaps this is a good one to introduce to our `js-utils` package. Perhaps a better option is to use an external library with tests that is more wide-spread and battle-tested? 

What are your thoughts?

Reply:
I was iffy on re-writing throttle by hand, as well. I would have _zero_ problems omitting this change from my PR and coming back to it later once we have a widely re-usable alternative 👍

Reply:
Sounds good to leave it out for now and address it when we decide to do it for the rest of the codebase 👍